### ISSUE TITLE
New line is added before comment if it exceeds the maximum line length

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#17]
```
// in
(*  dsffmdnf nfjnkjfsdnfjsadknfsdkjhfasnmsdfsdfkdfmasd fjsdf sdfjns dfsdjkf sdjkf sdjf sdjkf sfd fsd f*)
FUNCTION_BLOCK POU

// out 

(*  dsffmdnf nfjnkjfsdnfjsadknfsdkjhfasnmsdfsdfkdfmasd fjsdf sdfjns dfsdjkf sdjkf sdjf sdjkf sfd fsd f*)
FUNCTION_BLOCK POU

// expected
(*  dsffmdnf nfjnkjfsdnfjsadknfsdkjhfasnmsdfsdfkdfmasd fjsdf sdfjns dfsdjkf sdjkf sdjf sdjkf sfd fsd f*)
FUNCTION_BLOCK POU
```

STweep v2.0.0-beta
[settings.txt](https://github.com/Barteling/STweep/files/5879528/settings.txt)

**Comment by Barteling** [comment]
I think this is fixed in my master branch (not yet released), i'll check it, and fix it otherwise nonetheless.

**Comment by Barteling** [comment]
Fixed in release 2.0.1.
https://www.stweep.com/Releases/
